[2023-03-12 15:50:21,205] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: attack_dag.trigger_defence manual__2023-03-12T15:21:28.217282+00:00 [queued]>
[2023-03-12 15:50:21,232] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: attack_dag.trigger_defence manual__2023-03-12T15:21:28.217282+00:00 [queued]>
[2023-03-12 15:50:21,233] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-12 15:50:21,235] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-12 15:50:21,236] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-12 15:50:21,272] {taskinstance.py:1377} INFO - Executing <Task(TriggerDagRunOperator): trigger_defence> on 2023-03-12 15:21:28.217282+00:00
[2023-03-12 15:50:21,280] {standard_task_runner.py:52} INFO - Started process 64 to run task
[2023-03-12 15:50:21,287] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'attack_dag', 'trigger_defence', 'manual__2023-03-12T15:21:28.217282+00:00', '--job-id', '569', '--raw', '--subdir', 'DAGS_FOLDER/attack.py', '--cfg-path', '/tmp/tmp_bfhtbv8', '--error-file', '/tmp/tmp3ae0razb']
[2023-03-12 15:50:21,289] {standard_task_runner.py:80} INFO - Job 569: Subtask trigger_defence
[2023-03-12 15:50:21,430] {task_command.py:369} INFO - Running <TaskInstance: attack_dag.trigger_defence manual__2023-03-12T15:21:28.217282+00:00 [running]> on host 596adcca7378
[2023-03-12 15:50:21,619] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=attack_dag
AIRFLOW_CTX_TASK_ID=trigger_defence
AIRFLOW_CTX_EXECUTION_DATE=2023-03-12T15:21:28.217282+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-12T15:21:28.217282+00:00
[2023-03-12 15:50:23,964] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=attack_dag, task_id=trigger_defence, execution_date=20230312T152128, start_date=20230312T155021, end_date=20230312T155023
[2023-03-12 15:50:24,033] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-12 15:50:24,103] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
